src/components/Cards/ImageCard/ImageCard.module.scss

```diff
@@ -55,7 +55,7 @@ $placeholder-background: #000;
 		position: absolute !important;
 		top: 0;
 		left: 0;
-		height: auto !important;
+		height: 100% !important;
```

Comment:
Yeah nice this does make the image position centred on the preview page. Tricky handling the different image heights in grid view across all domains. Looks good 👍